tests/bootstrap.php

```diff
@@ -139,6 +139,3 @@
 // does not allow the sessionid to be set after stdout
 // has been written to.
 session_id('cli');
-
-// Fix multiple http/server requests in a single test method.
-$_SERVER['PHP_SELF'] = '/';
```

Comment:
Why was this removed?

Reply:
removed in other PR

Reply:
@dereuromark I did not do it. Must be one of thouse side effects of rebasing a branch in Git.